src/docs/sdk/event-payloads/properties/measurements.mdx

```diff
@@ -0,0 +1,43 @@
+`measurements`
+
+: _Optional_. An object containing standard/custom measurements with keys signifying the name of the measurement.
+
+Standard measurement keys currently supported are from the following list taken from [here](https://github.com/getsentry/sentry/blob/a8c960a933d2ded5225841573d8fc426a482ca9c/static/app/utils/discover/fields.tsx#L654-L676).
+
+```json
+[
+  // web
+  "fp",
+  "fcp",
+  "lcp",
+  "fid",
+  "cls",
+  "ttfb",
+  "ttfb.requesttime",
+  // mobile
+  "app_start_cold",
+  "app_start_warm",
+  "frames_total",
+  "frames_slow",
+  "frames_frozen",
+  "frames_slow_rate",
+  "frames_frozen_rate",
+  "stall_count",
+  "stall_total_time",
+  "stall_longest_time",
+  "stall_percentage"
```

Comment:
I didn't find the usage of this one on RN.

Reply:
It's referenced by Sentry: https://github.com/getsentry/sentry/blob/7447331d8e02031d585bd0c0aec76ed1334b9bde/src/sentry/search/events/constants.py#L32

Couldn't find it in Relay, not sure how this works. Maybe @untitaker can help.

Reply:
One of the open questions is if relay 'knows' the difference between standard/custom measurements or if this is only reflected in the sentry codebase.

Reply:
Regarding `stall_*` measurements: I searched through all of getsentry, and it looks like they were _only_ defined in Sentry. Later on, we pulled them into Relay and the ingest load tester. If they are not emitted by any SDK, we should remove them.

Regarding units: Relay will have to know about units of the past built-in measurements for backwards compatibility. For new measurements, and also going forward for all SDKs, it would be better to declare all units on the client and send them in to avoid mismatches and outdated lists.

Reply:
if I understood @jan-auer correctly, relay will have to know existing/standard measurements to infer their units.

currently only sentry has such a list, whether that list moves to Relay is tbd (we currently send it from sentry to relay via projectconfig)

Reply:
stall measurements are sent by the [RN SDK](https://github.com/getsentry/sentry-react-native/blob/main/src/js/tracing/stalltracking.ts)

Reply:
Thanks, and the rates / percentages are defined in Sentry so they don't have to be added to the public interface:
https://github.com/getsentry/sentry/blob/1c4ee6be215dadbf9bec19b3b428464a1dfe05b3/src/sentry/search/events/fields.py#L380-L418

Reply:
Actually, I'm wondering whether we should mark them as *reserved* and then never use them. They'll be defined as derived metrics in Sentry and can never be ingested.

Reply:
Let's not do any special casing for the three computed "measurements", but document them as reserved. That means, we should not reject the names in SDKs or during ingestion, but we should let people know that the name is used in Sentry.

There can be multiple metrics with the same name, and they are disambiguated in the API by type, so in the product we'll be fine. I'm still working out what that would mean for Discover though.

@wmak sharing a thought here that we can follow up on somewhere else: We should evaluate renaming those three specific measurements:
- Two are called `*_rate` like everything else in the product, but one is called `_percentage`. As opposed to functions like `failure_rate()` they _can_ be computed within a single transaction though, if I'm not mistaken?
- They share the same namespace with ingested measurements but they are not actually ingested. I wonder if they should actually be materialized in Relay.

Reply:
They can be computed within a single transaction :+1:
Is it possible to materialize these in relay if they're going to be a metric as well? Would simplify the product a bit

Reply:
Yes, in that case we can do the division in Relay, write them back into the measurements array, before extracting metrics for all measurements including the computed ones. I'd have to think a bit about migrating this forward, for a while we should probably keep the computation in discover around until we've reached 90d of history.